scripts/yocto-build-and-test.sh

```diff
@@ -121,6 +121,13 @@ prepare_build_config() {
 LICENSE_FLAGS_WHITELIST = "commercial_mender-binary-delta"
 FILESEXTRAPATHS_prepend_pn-mender-binary-delta := "${WORKSPACE}/mender-binary-delta:"
 PREFERRED_VERSION_pn-mender-binary-delta = "$mender_binary_delta_version"
+EOF
+
+    cat >> $BUILDDIR/conf/local.conf <<EOF
+LICENSE_FLAGS_WHITELIST += "commercial_monitor-client"
+FILESEXTRAPATHS_prepend_pn-monitor-client := "${WORKSPACE}:"
```

Comment:
As a rare exception to an otherwise good rule: Could you use `$WORKSPACE` instead of `${WORKSPACE}`? This *will* expand correctly as is, but it's confusing to read, because it looks like Bitbake syntax, and the `WORKSPACE` variable has a different meaning inside Bitbake.